shell/common/platform_view_service_protocol.cc

```diff
@@ -314,4 +316,29 @@ void PlatformViewServiceProtocol::ScreenshotGpuTask(SkBitmap* bitmap) {
   canvas->flush();
 }
 
+const char* PlatformViewServiceProtocol::kWaitUIThreadIdleExtensionName =
+    "_flutter.waitUIThreadIdle";
```

Comment:
I misunderstood the meaning of Tasks, you are right. It will wait for all the previous tasks to be flushed.